Kickstarter-iOS/Views/Cells/PledgeDescriptionCell.swift

```diff
@@ -0,0 +1,155 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+final class PledgeDescriptionCell: UITableViewCell, ValueCell {
+  // MARK: - Properties
+
+  private lazy var rootStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var containerImageView: UIView = { UIView.init() }()
+  private lazy var pledgeImage: UIImageView = { UIImageView(frame: .zero) }()
+  private lazy var descriptionStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var estimatedDeliveryLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var dateLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var descriptionLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var learnMoreLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var spacerView: UIView = { UIView(frame: .zero) }()
+
+  func configureWith(value: String) {
+    _ = self.dateLabel
+      |> \.text .~ value
+  }
+
+  // MARK: - Lifecycle
+
+  override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+    super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+    _ = (self.rootStackView, self.contentView)
+      |> ksr_addSubviewToParent()
+      |> ksr_constrainViewToEdgesInParent()
+
+    self.containerImageView.widthAnchor.constraint(equalToConstant: 100.0).isActive = true
```

Comment:
Just a suggestion on a nice way to group the activation of constraints:

```swift
self.rootStackView.addArrangedSubview(self.containerImageView)
self.containerImageView.addSubview(self.pledgeImage)

NSLayoutConstraint.activate([
  self.containerImageView.widthAnchor.constraint(equalToConstant: 100.0),
  self.pledgeImage.widthAnchor.constraint(equalToConstant: 90.0),
  self.pledgeImage.heightAnchor.constraint(equalToConstant: 130.0),
  self.pledgeImage.centerXAnchor.constraint(equalTo: self.containerImageView.centerXAnchor)
])
```

Either is fine!

Reply:
Note that your subviews must be added to their parents/stackviews before activating the constraints.